packages/open-next/src/build/patch/patches/patchFetchCacheISR.ts

```diff
@@ -111,3 +134,20 @@ export const patchUnstableCacheForISR: CodePatcher = {
     },
   ],
 };
+
+export const patchUseCacheForISR: CodePatcher = {
+  name: "patch-use-cache-for-isr",
+  patches: [
+    {
+      versions: ">=15.3.0",
+      field: {
+        pathFilter: getCrossPlatformPathRegex(
+          String.raw`(server/chunks/.*\.js|.*\.runtime\..*\.js|use-cache/use-cache-wrapper\.js)$`,
```

Comment:
nit: leading `.*` should not be needed

```suggestion
          String.raw`(server/chunks/.*\.js|\.runtime\..*\.js|use-cache/use-cache-wrapper\.js)$`,
```